### PR TITLE
feat(frontend): implement NoTokensPlaceholder

### PR DESCRIPTION
**Pull request by loki344**
# Motivation

If a empty token list is displayed, we want a better placeholder to guide the user.

# Changes

- Implement component for NoTokensPlaceholder and use it.
- Adjust texts

# Note
BTC logo fix: https://github.com/dfinity/oisy-wallet/pull/3235

# Tests

Before:
image

After:
320px:
image

600px:
image

960px:
image

1280px:
image

1600px:
image

1920px:
image

### CODE REVIEW

File: src/frontend/src/lib/components/tokens/NoTokensPlaceholder.svelte [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+<div class="py-12">
+	<div class="mb-5 flex justify-center p-2">
+		<span class="-mr-4">
+			<Logo color="white" ring size="lg" src={icpDark}></Logo>
+		</span>
+		<span class="-mr-4">
+			<Logo color="white" ring size="lg" src={ETHEREUM_TOKEN.icon}></Logo>
+		</span>
+		<Logo color="white" ring size="lg" src={BTC_MAINNET_TOKEN.icon}></Logo>
```
Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/lib/components/tokens/NoTokensPlaceholder.svelte:17]: for consistency, i would use either all assets (icpDark.svg, bitcoin.svg, etc), or all TOKEN icons, or all NETWORK icons

File: src/frontend/src/lib/components/tokens/NoTokensPlaceholder.svelte [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+	import icpDark from '$eth/assets/icp_dark.svg';
+	import bitcoin from '$icp/assets/bitcoin.svg';
+	import eth from '$icp-eth/assets/eth.svg';
+	import Logo from '$lib/components/ui/Logo.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+</script>
+
+<div class="py-12">
```
Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/lib/components/tokens/NoTokensPlaceholder.svelte:9]: Forgot this in my last review, sorry: a transition here maybe? how does it behaves on transition?

Comment (loki344) [reply src/frontend/src/lib/components/tokens/NoTokensPlaceholder.svelte:9]: I believe the Logo component already takes care of this: `<div
	class="flex items-center justify-center overflow-hidden rounded-full ring-white"
	class:bg-dust={color === 'dust' && !loaded}
	class:bg-off-white={color === 'off-white' && !loaded}
	class:bg-white={color === 'white' && !loaded}
	class:opacity-10={!loaded}
	class:ring-2={ring}
	style={`width: ${sizePx}; height: ${sizePx}; transition: opacity 0.15s ease-in;`}
>`

But.. good point about the loading time. This leads to a undesired behaviour:
https://github.com/user-attachments/assets/f76c88a1-54ef-4c3b-bbf2-58ad0d5ffed4

I added very small z-indexes to ensure the stacking. Here's a quick video, also ensuring modal functionality still works.
https://github.com/user-attachments/assets/7fda8d00-f65d-4fb6-9f9f-8fafe1539973

**Review by AntonioVentilii-DFINITY — APPROVED** [review]
LGTM tks

**Pull request merged**